docs/docs/docs_es/basic-tutorial/performance.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Bonus: Rendimiento'
+---
+
+Nuestra implementación actual es perfectamente valida, pero hay algunos puntos importantes a considerar relacionados con el rendimiento. Desde que nuestra aplicación evolucionará desde ser un proyecto pequeño hasta convertirse en un programa corporativo de millones de líneas de código.
+
+Pensemos en que causará que cada uno de nuestro componentes se renderice:
+
+### `<TodoList />`
+
+Este componente esta suscrito a `filteredTodoListState`, que es un selector que tiene dependencia con `todoListState` y `todoListFilterState`. Esto significa que `TodoList` volverá a renderizar cuando los siguientes estados cambien:
+
+- `todoListState`
+- `todoListFilterState`
+
+### `<TodoItem />`
+
+Este componente esta suscrito a `todoListState`, entonces se volverá a renderizar cada vez que `todoListState` cambie, y también cuando el componente padre, `TodoList`, renderice.
+
+### `<TodoItemCreator />`
+
+Este componente no esta suscrito al estado de Recoil (`useSetRecoilState()` no crea alguna subscripción), por lo tanto, solo renderizará cuando su componente padre lo haga.
+
+### `<TodoListFilters />`
+
+Este componente esta suscrito a `todoListFilterState`, entonces se volverá a renderizar cada vez que ese estado cambie, y también cuando el componente padre, `TodoList`, renderice.
+
+
+### `<TodoListStats />`
+
+Este componente esta suscrito a `filteredTodoListState`, entonces se volverá a renderizar cada vez que ese estado cambie, y también cuando el componente padre, `TodoList`, renderice.
+
+## Oportunidad de mejoras
+
+La implementación actual tiene algunos inconvenientes, principalmente por el hecho que estamos renderizando el árbol entero cada vez que hacemos cambios al `todoListState`. Y debido a que `<TodoList />` es el padre de todos nuestros componentes, cuando éste vuelva a renderizar, también todos sus hijos lo harán.
+
+Idealmente, los components deberían de renderizar solo cuando es absolutamente necesario (cuando los datos que los componentes despliegan ha cambiado).
```

Comment:
despliegan->emplean/utilizan, muestran is not 100% accurate, and I don’t understand the meaning of despliegan in this context.